packages/github/webhook.js

```diff
@@ -52,7 +59,7 @@ function main(params) {
   var whiskCallbackUrl = urlHost.protocol + '//' + process.env.__OW_API_KEY + "@" + urlHost.host + '/api/v1/namespaces/' + encodeURIComponent(triggerName[1]) + '/triggers/' + encodeURIComponent(triggerName[2]);
 
   // The URL to create the webhook on Github
-  var registrationEndpoint = 'https://api.github.com/repos/' + (organization ? organization : username) + '/' + repository + '/hooks';
+  var registrationEndpoint = 'https://'+ githubApi +'/repos/' + (organization ? organization : username) + '/' + repository + '/hooks';
```

Comment:
How about including a protocol into the variable too?
In some cases, the protocol might need to be configurable.

Reply:
I have inserted in the variable also the protocol